src/main/java/io/github/nucleuspowered/nucleus/modules/item/commands/lore/LoreSetBaseCommand.java

```diff
@@ -26,10 +26,12 @@
     @Inject private MessageProvider provider;
 
     final String loreKey = "lore";
+    final String loreLine = "line";
 
     @Override
     public CommandElement[] getArguments() {
         return new CommandElement[] {
+                GenericArguments.optionalWeak(GenericArguments.integer(Text.of(loreLine))),
```

Comment:
I know you have this as `optionalWeak`, but you're potentially breaking the `/lore add` and `/lore set` commands here, if players want a number at the beginning of their command.

I suggest spinning out your changes into a different base class instead, either extending from this one, or (based on what I see) just a different base class - then the integer (which can also use the `PositiveIntegerArgument`) should be non-optional.